Mobile/Android/Templates/471-reliable_uploads_actions_uploads_view.md

```diff
@@ -0,0 +1,64 @@
+### Uploads view
+
+#### Pr: https://github.com/owncloud/android/issues/471
+
+Devices:
+
+Server version:
+
+<br>
+ 
+Test Case ID | Test case     | Steps   | Expected behavior | Result | Related Comments
+|:----:|:------------- |:---------------|:-------------|-------------:|----------:
+|**Views**|||||
+|1| New option in menu | 1. Swipe to display the main menu <br> 2. Change orientation of device | 1. An new option "Uploads" is displayed in the menu<br> 2. The view is correct in both orientations|||
+|2| Upload menu |  1. Swipe to display the main menu<br> 2. Tap on Uploads <br> 3. Change orientation of device| 2. Three sections are displayed: current , failed and uploaded. Also there are two options to clear: all and failed.<br> 3. The view is correct in both orientations|||
+|**Uploads**||||
+|3| Upload a file (copy)| 1. Tap to upload a file<br> 2. Select a file to upload choosing "copy"| 2. The file appears on menu "Current" while is uploaded and in "Uploaded" when the upload is finished (change orientation). File is correctly copied (remains in origin)
+|4| Upload a file (move)| 1. Tap to upload a file<br> 2. Select a file to upload choosing "move" | 2. The file appears on menu "Current" while is uploaded and in "Uploaded" when the upload is finished (change orientation). File is correctly moved (deleted in origin)
```

Comment:
I'd also check the progress bar, on the one hand that it´s shown when the upload is on progress and that it's removed when the upload finishes.
Check, there has been some changes on code: https://github.com/owncloud/android/commit/2c4daf3a93bc790e618a47ec24bf3db84f320f20 and https://github.com/owncloud/android/commit/6f58527273a93fd9fea033a46cad839c565e1357
Plus, if the upload is cancel, it´s removed..
thx